tests/tools/onert_train/src/nnfw_util.cc

```diff
@@ -112,4 +112,9 @@ std::ostream &operator<<(std::ostream &os, const nnfw_train_info &info)
   return os;
 }
 
+template <typename E> constexpr auto to_underlying(E e) noexcept
+{
+  return static_cast<std::underlying_type_t<E>>(e);
+}
+
```

Comment:
You don't need this function here, do you?

Reply:
Ah, Yess. 
My mistake. I'll remove it!